test/tbb/test_task_group.cpp

```diff
@@ -1173,6 +1173,23 @@ TEST_CASE("Task handle run"){
     CHECK_MESSAGE(h == nullptr, "Delayed task can be executed only once");
 }
 
+//! Basic test for task handle
+//! \brief \ref interface \ref requirement
+TEST_CASE("Task handle run_and_wait"){
+    tbb::task_handle h;
+
+    tbb::task_group tg;
+    std::atomic<bool> run {false};
```

Comment:
It will help TSAN to check that `defer` does not spawn the task.
```suggestion
    bool run {false};
```